process/ccla_template_aswf2020.md

```diff
@@ -0,0 +1,160 @@
+    license. You represent further that the employee of the Corporation
+    designated as the Initial CLA Manager below (and each who is
+    designated in a subsequent written modification to the list of CLA
+    Managers) (each, a "CLA Manager") is authorized to maintain (1) the
+    list of employees of the Corporation who are authorized to submit
+    Contributions on behalf of the Corporation, and (2) the list of CLA
+    Managers; in each case, using the designated system for managing
+    such lists (the "CLA Tool").
+
+5.  You represent that each of Your Contributions is Your original
+    creation (see section 7 for submissions on behalf of others).
+
+6.  You are not expected to provide support for Your Contributions,
+    except to the extent You desire to provide support. You may provide
+    support for free, for a fee, or not at all. Unless required by
+    applicable law or agreed to in writing, You provide Your
+    Contributions on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+    OF ANY KIND, either express or implied, including, without
+    limitation, any warranties or conditions of TITLE, NON-INFRINGEMENT,
+    MERCHANTABILITY, or FITNESS FOR A PARTICULAR PURPOSE.
+
+7.  Should You wish to submit work that is not Your original creation,
+    You may submit it to the Project separately from any Contribution,
+    identifying the complete details of its source and of any license or
+    other restriction (including, but not limited to, related patents,
+    trademarks, and license agreements) of which you are personally
+    aware, and conspicuously marking the work as "Submitted on behalf of
+    a third-party: \[named here\]".
+
+8.  It is your responsibility to use the CLA Tool when any change is
+    required to the list of designated employees authorized to submit
+    Contributions on behalf of the Corporation, or to the list of the
+    CLA Managers.
+
+\[Please complete and sign on the next page.\]
+
+::: {style="page-break-after: always; visibility: hidden"}
+```{=tex}
+\pagebreak
+```
+:::
+
+Please sign:
+\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_ Date:
+\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_
+
+Signatory Name:
+\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_
+
+Signatory E-mail:
+\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_
+
+Signatory Title:
+\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_
+
+Corporation Name:
+\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_
+
+Corporation Address:
+\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_
+
+------------------------------------------------------------------------
+
+------------------------------------------------------------------------
```

Comment:
In lines 152 and 154, I think Markdown is rendering these as horizontal rules / line breaks, guessing they should probably get the escape-slashes instead?

Reply:
Good catch! Thanks :-)